File: BroadcastTest2/app/src/main/java/com/voidsoul/broadcasttest2/MainActivity.java
package com.voidsoul.broadcasttest2;

import androidx.appcompat.app.AppCompatActivity;

import android.content.IntentFilter;
import android.os.Bundle;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {
    private TextView textView;
    private AnotherBroadcastReceiver anotherBroadcastReceiver;
    @Override
    protected void onDestroy() {
        super.onDestroy();
        unregisterReceiver(anotherBroadcastReceiver);
    }

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        textView = (TextView) findViewById(R.id.textview111);
        IntentFilter intentFilter = new IntentFilter();
        anotherBroadcastReceiver = new AnotherBroadcastReceiver();
        intentFilter.addAction("com.voidsoul.broadcasttest.MY_BROADCAST");

//        intentFilter.setPriority(100);
        registerReceiver(anotherBroadcastReceiver, intentFilter);
    }
}

File: testcamerax/app/src/main/java/com/voidsoul/testcamerax/MainActivity.kt
package com.voidsoul.testcamerax

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.camera.core.CameraSelector
import androidx.camera.lifecycle.ProcessCameraProvider
import androidx.core.content.ContextCompat
import com.voidsoul.testcamerax.R

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    private fun startCamera(){
        val cameraProviderFuture = ProcessCameraProvider.getInstance(this)
        val cameraSelector = CameraSelector.Builder().requireLensFacing(CameraSelector.LENS_FACING_BACK).build()
        cameraProviderFuture.addListener({
            val cameraProvider = cameraProviderFuture.get()
            cameraProvider.bindToLifecycle(this, cameraSelector)
        }, ContextCompat.getMainExecutor(this))
    }


}